tests/testTriggers.go

```diff
@@ -2584,14 +2584,16 @@ func TriggerRestartKvdbVolDriver(contexts *[]*scheduler.Context, recordChan *cha
 		}
 		err = Inst().S.RefreshNodeRegistry()
 		UpdateOutcome(event, err)
+		err = Inst().V.RefreshDriverEndpoints()
```

Comment:
RefreshDriverEndpoints helps populate volume driver details of the node registry.